README.md

```diff
@@ -51,7 +51,7 @@ Or, if you customized the installation prefix:
 
 ```
 PREFIX=/thing
-rm "${PREFIX}"/bin/kubectl-crossplane-stack-*
+rm "${PREFIX}"/bin/kubectl-crossplane-package-*
```

Comment:
might want to keep the old command as well for uninstalling older versions

Reply:
Good idea! Will update :)